CHANGELOG.txt

```diff
@@ -1,5 +1,11 @@
-4.0.2
+4.1.0
+* Exposes billing metrics
+* Adds runtime environment headers for http fauna requests
+* Always sends query timeout header
+* Enables user to pass a custom user agent
```

Comment:
Is there an example demonstrating how to do this?

Reply:
It can be added to FaunaClient builder as other parameters:
```
FaunaClient adminClient =
    FaunaClient.builder()
            .withEndpoint("fauna-endpoint")
            .withSecret("fauna-secret")
            .withUserAgent("my-user-agent")
            .build();
```